source/datasets.rst

```diff
@@ -197,12 +197,14 @@ data by using a browser:
 
 #. Access the :strong:`Swagger interface` of the datasets in the
    Mobility domain, located at
-   https://mobility.api.opendatahub.bz.it/v2/swagger-ui.html. Like in
-   the case of the tourism' Swagger interface, you can learn REST API
-   call for that domain and fetch data for your application. There is
-   a dedicated howto to learn more how to interact with this
-   interface: ref:`get-started-mobility`
-
+   https://swagger.opendatahub.bz.it/?url=https://mobility.api.opendatahub.bz.it/v2/apispec. Like
+   in the case of the tourism' Swagger interface, you can learn REST
+   API call for that domain and fetch data for your application. There
+   is a dedicated howto to learn more how to interact with this
+   interface: ref:`get-started-mobility`. More possibility to interact
+   with the Mobility domain datasets are described in the
+   :ref:`dedicated howto <get-started-mobility>`
+	
 #. Open the :strong:`Analytics for Mobility` web page, at
    https://analytics.mobility.bz.it/. This portal uses data in the
```

Comment:
The new URL is https://analytics.opendatahub.bz.it

Reply:
This has nothing to do with this issue, better open a new one IMHO.

Reply:
OK, but lets find a compromise between to many small issues and just being efficient in tackling errors. Otherwise, we need just for one wrong URL: a new issue, a new PR, a new review, a new merge... that can be a lot of effort for a little bugfix